src/main/jbake/content/download-ng.adoc

```diff
@@ -7,12 +7,18 @@
 
 [.table.table-bordered,options="header"]
 
+== Release Integrity
+
+You **must** link:https://www.apache.org/info/verification.html[verify] the integrity of the downloaded files. We provide OpenPGP signatures  (*.asc files) for every release file. This signature should be matched against link:https://downloads.apache.org/tomee/KEYS[KEYS] file which contains the OpenPGP keys of TomEE's Release Managers. We also provide SHA-512 checksums for every release file. After you download the file, you should calculate a checksum for your download, and make sure it is the same as ours.
+
+== Download Links
+
 |===
 |Name|Version|Date|Size|Type|Links
-|TomEE plume|9.0.0-M3|25 Jan 2021|65 MB |ZIP| https://www.apache.org/dyn/closer.cgi/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-plume.zip[icon:download[] ZIP] https://www.apache.org/dist/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-plume.zip.sha256[icon:download[] SHA256] https://www.apache.org/dist/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-plume.zip.sha512[icon:download[] SHA512]
-|TomEE plus|9.0.0-M3|25 Jan 2021|58 MB |ZIP| https://www.apache.org/dyn/closer.cgi/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-plus.zip[icon:download[] ZIP] https://www.apache.org/dist/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-plus.zip.sha256[icon:download[] SHA256] https://www.apache.org/dist/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-plus.zip.sha512[icon:download[] SHA512]
-|TomEE webprofile|9.0.0-M3|25 Jan 2021|41 MB |ZIP| https://www.apache.org/dyn/closer.cgi/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-webprofile.zip[icon:download[] ZIP] https://www.apache.org/dist/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-webprofile.zip.sha256[icon:download[] SHA256] https://www.apache.org/dist/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-webprofile.zip.sha512[icon:download[] SHA512]
-|TomEE microprofile|9.0.0-M3|25 Jan 2021|41 MB |ZIP| https://www.apache.org/dyn/closer.cgi/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-microprofile.zip[icon:download[] ZIP] https://www.apache.org/dist/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-microprofile.zip.sha256[icon:download[] SHA256] https://www.apache.org/dist/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-microprofile.zip.sha512[icon:download[] SHA512]
+|TomEE plume|9.0.0-M3|25 Jan 2021|65 MB |ZIP| https://www.apache.org/dyn/closer.cgi/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-plume.zip[icon:download[] ZIP] https://www.apache.org/dist/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-plume.zip.sha256[icon:download[] SHA256] https://www.apache.org/dist/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-plume.zip.sha512[icon:download[] SHA512] https://www.apache.org/dist/tomee/tomee-9.0.0-M3/apache-tomee-9.0.0-M3-plume.zip.asc[icon:download[] PGP]
```

Comment:
is it a manual fix? Should go into https://github.com/apache/tomee-site-generator/blob/master/src/main/java/org/apache/tomee/website/Downloads.java#L152 probably

Reply:
@rmannibucau Yes indeed. Thx for the hint. I will update the Java code as well